Zend/zend_gc.c

```diff
@@ -147,6 +147,16 @@ static void gc_trace_ref(zend_refcounted *ref) {
 }
 #endif
 
+typedef struct _gc_refcounted_stack_element {
```

Comment:
`zend_refcounted` can be used directly instead of wrapping in `gc_refcounted_stack_element`

Reply:
done